lib/main.dart

```diff
@@ -104,3 +111,23 @@ class _MyAppState extends State<MyApp> {
     );
   }
 }
+
+// TODO REMOVE THIS. ONLY DEBUGGING PURPOSES
+class LoggerLink extends Link {
+  @override
+  Stream<Response> request(
+    Request request, [
+    NextLink? forward,
+  ]) {
+    Stream<Response> response = forward!(request).map((Response fetchResult) {
+      print("Request: " + request.toString());
+      return fetchResult;
+    }).handleError((error) {
+      // throw error;
+    });
+
+    return response;
+  }
+
+  LoggerLink();
+}
```

Comment:
Not a fan of merging as is, but will refactor this to be able to add a flag for development (hopefully after this one)